### ISSUE TITLE
Publishing on the PyPI

### ISSUE THREAD

**Issue by isidentical** [we-like-parsers/pegen#254]
Pegen's Python parser generator is really cool, and also very [useful](https://github.com/isidentical/pyasdl/blob/master/pyasdl/parser.py) though the version on the PyPI seems not sync with the upstream version under Tools/peg_generator. It is possible now to just clone CPython and install it though not very feasible. Or the other way would be, after generating the parser just hosting `pegen/parser.py` / `pegen/tokenizer.py` under the project (which is what I do now) though it is not a very good solution either. I think it would be really amazing if we host this package on the PyPI (maybe with automating some stuff on the GH actions to clone CPython, check if anything changed under `Tools/peg_generator`, and sync it with here and open an issue to notify the maintainers that they should make a release). If any help needed, I can try to assist.

**Comment by pablogsal** [comment]
This is indeed some of the action items from the core Dev sprint. With all that has been happening it has been impossible to find time for it but I will try to do it soon.

One limitation is that the C generator is not that useful outside C Python because the AST symbols are not exposed, so we can only expose the Python generator.

**Comment by lysnikolaou** [comment]
I agree that exposing only the Python generator is the way to go. It's both less work maintaining it for us and the value of exposing C generator is really really minimal.

**Comment by lysnikolaou** [comment]
Should we maybe start a new repository and move the version we want to publish there? With all the blog post code and the C generator code here, it feels that a new repo would be much cleaner.

**Comment by isidentical** [comment]
I think it makes sense (to preserve the blog posts / sources)

**Comment by pablogsal** [comment]
> Should we maybe start a new repository and move the version we want to publish there? With all the blog post code and the C generator code here, it feels that a new repo would be much cleaner.

Appart from the blog post, is there any reason we want to preserve the rest of the code in this repo? The canonical version of all the code itself lives in CPython now and much if the stuff we have here currently is out of date.

**Comment by MatthieuDartiailh** [comment]
I would also be interested in the Python generator since I maintain a project that defines a superset of Python (https://github.com/nucleic/enaml). Currently it relies on ply which is LALR1 parser and may not be able to keep up with the new language features making use of the full power of the parser.
